godtools/App/Services/Renderer/Views/MobileContent/Views/ContentAnimation/MobileContentAnimationViewModel.swift

```diff
@@ -37,11 +47,38 @@ class MobileContentAnimationViewModel: MobileContentViewModel {
             case .failure( _):
                 animatedViewModel = nil
             }
+            
+            playbackState = animationModel.autoPlay ? .play : .stop
```

Comment:
unfortunately `playbackState` may be a bit more complex then this. If the animation has `loop=false` it should automatically transition from .play to .stop once the animation finishes

Reply:
I know the lottie APIs on Android allow inspection to see if the animation is currently playing or not